src/libs/actions/PersonalDetails.js

```diff
@@ -82,32 +83,37 @@ function getMaxCharacterError(isError) {
  * @return {Object}
  */
 function formatPersonalDetails(personalDetailsList) {
-    return _.reduce(personalDetailsList, (finalObject, personalDetailsResponse, login) => {
+    Timing.start(CONST.TIMING.PERSONAL_DETAILS_FORMATTED);
+    const formattedResult = {};
+
+    // This method needs to be SUPER PERFORMANT because it can be called with a massive list of logins depending on the policies that someone belongs to
+    // eslint-disable-next-line rulesdir/prefer-underscore-method
```

Comment:
Is there a reason we are not using the underscore method? `_.each` provides (value, key) as arguments to the callback function.

Reply:
Yeah, so that it can be "SUPER PERFORMANT" 😁  If you research iterating object details, this vanilla way is typically one of the fastest.

Reply:
Makes sense! Thanks for clarifying!